.github/workflows/pull-request.yaml

```
@@ -54,35 +54,49 @@ jobs:
+            if [[ "$file" =~ ^\.github/ ]] || \
+               [[ "$file" =~ ^\.vscode/ ]] || \
+               [[ "$file" =~ ^docs/ ]] || \
+               [[ "$file" =~ ^README ]] || \
+               [[ "$file" =~ ^LICENSE ]] || \
+               [[ "$file" =~ ^\.(gitignore|env)$ ]] || \
+               [[ "$file" == "package.json" ]] || \
+               [[ "$file" == "package-lock.json" ]] || \
+               [[ "$file" == "yarn.lock" ]]; then
+              continue
+            fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)